schemes/rrtmgp/rrtmgp_inputs.F90

```diff
@@ -26,10 +27,9 @@ subroutine rrtmgp_inputs_init(ktopcam, ktoprad, nlaycam, sw_low_bounds, sw_high_
      integer,                         intent(in) :: nradgas                ! Number of radiatively active gases
      integer,                         intent(in) :: pverp                  ! Number of vertical interfaces
      integer,                         intent(in) :: pver                   ! Number of vertical layers
-     integer,                         intent(in) :: iradsw                 ! Freq. of shortwave radiation calc in time steps (positive) or hours (negative).
+     integer,                         intent(in) :: iradsw                 ! Freq. of shortwave radiation calc in time steps                                                                             ! (positive) or hours (negative).
```

Comment:
What happened to the comment here?

Reply:
Good catch.  There actually wasn't a new line there!